lib/handler/configurator/proxy.c

```diff
@@ -142,6 +149,71 @@ static int on_config_ssl_cafile(h2o_configurator_command_t *cmd, h2o_configurato
     return ret;
 }
 
+static h2o_cache_t *create_ssl_session_cache(uint64_t capacity, unsigned lifetime)
```

Comment:
Type of `capacity` should match that of `h2o_cache_create` (and generally speaking, capacity is bound to by the maximum size of memory that can be allocated, thus should be `size_t`).